fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/domain/LoanProductConfigurableAttributes.java

```diff
@@ -18,19 +18,18 @@
  */
 package org.apache.fineract.portfolio.loanproduct.domain;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.JoinColumn;
-import javax.persistence.ManyToOne;
-import javax.persistence.Table;
-
+import org.apache.commons.lang.builder.EqualsBuilder;
```

Comment:
as above, why use Commons Lang here but java.lang.Objects helpers everywhere else? The Commons Lang helpers are less efficient, because they create intermediate very short live objects (to be GC'd), which can be easily avoided.